web/ui/react-app/src/pages/rules/Rules.tsx

```diff
@@ -1,15 +1,147 @@
 import React, { FC } from 'react';
 import { RouteComponentProps } from '@reach/router';
 import PathPrefixProps from '../../types/PathPrefixProps';
-import { Alert } from 'reactstrap';
-
-const Rules: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => (
-  <>
-    <h2>Rules</h2>
-    <Alert color="warning">
-      This page is still under construction. Please try it in the <a href={`${pathPrefix}/rules`}>Classic UI</a>.
-    </Alert>
-  </>
-);
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { Alert, Table } from 'reactstrap';
+import { useFetch } from '../../hooks/useFetch';
+import { Link } from '@reach/router';
+import { Rule } from '../../types/types';
+import { formatRelative, roundUp } from '../../utils';
+import { now } from 'moment';
+
+interface RuleGroup {
+  name: string;
+  file: string;
+  rules: Rule[];
+  evaluationTime: string;
+  lastEvaluation: string;
+}
+
+interface RulesMap {
+  groups: RuleGroup[];
+}
+
+interface MapProps {
+  map: Record<string, string>;
+  term: string;
+}
+
+const AlertRuleMaps: FC<RouteComponentProps & MapProps> = ({ term, map }) => {
+  return (
+    <div>
+      <strong>{term}:</strong>
+      <div style={{ marginLeft: '3%' }}>
+        {Object.entries(map).map(([key, value], i) => (
+          <div key={i}>
+            {key}: {value}
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const Rules: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => {
+  const { response, error } = useFetch<RulesMap>(`${pathPrefix}/api/v1/rules`);
+
+  if (error) {
+    return (
+      <Alert color="danger">
+        <strong>Error:</strong> Error fetching Rules: {error.message}
+      </Alert>
+    );
+  } else if (response.data) {
+    const groups: RuleGroup[] = response.data.groups;
+    return (
+      <>
+        <h2>Rules</h2>
+        {groups.map((g, i) => {
+          return (
+            <Table size="sm" striped bordered key={i}>
```

Comment:
1. This table shouldn't be striped (old UI isn't either), instead, grey coloring should be used for pre-formatted rule cells (see old UI).

2. The size of this table should probably normal like in the old UI (so remove the `size="sm"` option), this also leads to nicer padding in the table cells.

Reply:
Done. Thanks.